samples/bluetooth/direct_test_mode/sysbuild/remote_shell/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
It looks like image for application core is not built when using sysbuild

Reply:
Yes, this does not work NCSDK-23805